### ISSUE TITLE
plugin-factory.umd.js Failed to load resource: the server responded with a status of 404 (Not Found)

### ISSUE THREAD

**Issue by briankitt** [Toxicable/module-loading#4]
Where should I get this module?  I don't know if it should have been included.  I started looking for a package in NPM, but not seeing anything out there.

Could you point me to the source for this module?

Thanks!

**Comment by Toxicable** [comment]
This isn't published as a npm package, it's more of a demo of what you can do with Angular if you want to go really far away from current build systems. I personalyl would not recommend it since it's not supported by Angular Core, instead I'd suggest doing this
https://www.youtube.com/watch?v=__H65AsA_bE&feature=youtu.be&t=2h14m13s

**Comment by briankitt** [comment]
Thanks!

But I'm not sure that approach is going to work.  I have a very peculiar
setup.  I will try to keep this rather generic so it doesn't get too wordy.

I have an angular app.  Due to a myriad of reason, each site will have
their own server and their own copy of the distributed main app.
There are a host of plugins they can use, if they have paid for the license
to those plugins.
So when I deliver the app, it doesn't come with all of the plugins, and
they can add more as they go, and we do not want to have to redistribute
the app to them.
Thus, the plugins have to be dynamic, but we do NOT want them to live on
the customer server, they must be hosted on a central server.  (this brings
Cors into play, which adds more complexity).

I don't make up the requirements, I'm stuck with implementing them.

Here is the process flow:
Angular app is set up on customer server, it is basically a bare bones
version of our app.
When it starts up, it will look at license info, and load up the
appropriate plugins.
These plugins are hosted at OUR server, not theirs.
The plugins on our server could either be stored as TypeScript or JS, I
don't care, whatever works is fine.  But on the customer side, they exist
only in memory.  If server goes down, they must be fetched from central
server gain.

I've been trying SystemJS, but can't get that to work. Your solutions looks
like a near perfect fit for what I am trying to do, but I can't get it to
work because the plugin umd file is missing.

Any ideas?
